src/App.vue

```diff
@@ -1,29 +1,41 @@
 <template>
   <div>
```

Comment:
```<div id='app'>```
to accomodate scoped styling which using # refers to id.
Otherwise flexbox will not be applied causing footer to float.

Reply:
Need this change so that footer will stick to bottom.

Reply:
Please fix linting errors: all empty lines need to be removed and fix indentation of html tags

Reply:
Okay cool, I will do thanks